projects/1.16/assets/pneumaticcraft-repressurized/pneumaticcraft/patchouli_books/book/zh_cn/entries/base_concepts/pressure.json

```
@@ -5,23 +5,23 @@
   "pages": [
     {
       "type": "text",
-      "text": "压缩空气是$(pncr)中的$(thing)能量系统/$，而这一系统是基于真实世界中依靠压力运转的机器的。$(li)压缩空气是由一系列称作$(thing)压缩机/$的机器$(italic)生产的/$，而这些压缩机有很多种。$(li)所有可被加压的机器都具有一项称为$(thing)容量/$（以mL计）的属性，代表在一个标准大气压（即$(ttcolor)$(t:该模组使用0bar表示标准大气压，即此时的*表压*)0bar/$）下其可存储的空气量。"
+      "text": "压缩空气是$(#228)气动工艺：再加压$()中的$(thing)能量系统/$，而这一系统是基于真实世界中依靠压力运转的机器的。$(li)压缩空气是由一系列称作$(thing)压缩机/$的机器$(italic)生产的/$，有很多种不同类型的压缩机可供选择。$(li)所有可被加压的机器都具有一项称为$(thing)容量/$（以mL计）的属性，代表在一个标准大气压所能容纳的气体体积（即$(ttcolor)$(t:本模组使用0bar表示标准大气压，即此时的*表压*)0bar/$）。"
     },
     {
       "type": "text",
-      "text": "$(li)一台机器所处的$(thing)压力 P/$可以以下列方法计算：$(p)  $(formula)P = (A / V) - 1/$$(p)其中$(formula)V/$代表$(thing)容量/$，$(formula)A/$代表该机器目前含有的$(thing)空气/$量。例：若一台机器的空气$(thing)容量/$为5000mL ，目前存储有20000mL$(thing)空气/$，则其此时的气压为$(thing)3bar/$。$(li)大多数机器的容量都可以使用$(l:base_concepts/upgrades#volume)容量升级/$提升，这意味着相等空气量变化导致的压力变化更小。"
+      "text": "$(li)机器或工具的$(thing)压力值P/$可以通过下列公式计算：$(p)  $(formula)P = (A / V) - 1/$$(p)其中$(formula)V/$代表$(thing)容量/$，$(formula)A/$代表该机器目前含有的$(thing)空气/$量。例：一台机器有5000mL大小的$(thing)容量/$，目前存储有20000mL$(thing)空气/$，则其此时的气压为$(thing)3bar/$。$(li)大多数机器的容量都可以使用$(l:base_concepts/upgrades#volume)容量升级/$提升，这意味着相等空气量变化导致的压力变化更小。"
```

```suggestion
      "text": "$(li)机器或工具的$(thing)压力值P/$可以通过下列公式计算：$(p)  $(formula)P = (A / V) - 1/$$(p)其中$(formula)V/$代表$(thing)容量/$，$(formula)A/$代表该机器目前含有的$(thing)空气/$量。例：一台机器5的$(thing)容量/$为5000mL，目前存储有20000mL$(thing)空气/$，则其此时的气压为$(thing)3bar/$。$(li)大多数机器的容量都可以使用$(l:base_concepts/upgrades#volume)容量升级/$提升，这意味着相等空气量变化导致的压力变化更小。"
```

容量算是个固定属性（虽然可以装升级），感觉这样还是好点。

---

你多了个5（），改的没毛病